scripts/CTP7_RX_opto_power.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+CTP7s=( eagle34 )
```

Comment:
How is this list used?
Should remove hard coding and use args if possible, or add a populate script as part of the rsync commands